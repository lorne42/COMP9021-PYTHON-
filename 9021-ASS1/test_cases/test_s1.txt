I have just seen Sirs Sanjay and Eleonore!
Sir Eve said: "At least one of Sir Sanjay and Sir Eleonore is a Knight."
Sir Frank said: "Exactly one of us is a Knave."
Who is a Knight and who is a Knave?
