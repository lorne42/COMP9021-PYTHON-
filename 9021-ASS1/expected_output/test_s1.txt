The Sirs are: Eleonore Eve Frank Sanjay
There are 5 solutions.